ops/dockerfiles/server.Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM mycro/solc-python:latest
+
+#setup uwsgi used in prod environments
+RUN apt-get update
+RUN pip3 install --upgrade pip
+RUN pip3 uninstall uwsgi
```

Comment:
sanity check on uninstall

Reply:
we need the below dependencies in order to ensure that uwsgi is installed correctly. If it's already installed without them we have to uninstall it first. I view this as defensive. As with the previous comments though if we're hitting image size issues we could remove it (just might make the dockerfile *slightly* more brittle in the future)

Reply:
Let's stay defensive 😃